features/account/customer_account/viewing_orders_history/viewing_order_recalculated_by_exchange_rate.feature

```diff
@@ -5,17 +5,16 @@ Feature: Viewing details of an order
     I want to be able to view details of my placed order
```

Comment:
This feature doesn't make sense, we should have a feature "Order is always placed in a base currency of a channel" and then in the scenario:
1. Browse shop in a different than base currency
2. Place order through UI
3. Verify than order amounts are in a base currency